docs/unmanaged_configuration.md

```diff
@@ -25,3 +25,50 @@ oc set env dc/fluentd MERGE_JSON_LOG=true
 * Possible log loss due to Elasticsearch rejecting documents due to inconsistent type mappings
 * Potential buffer storage leak caused by rejected message cycling
 * Overwrite of data for field with same names
+
+## ElasticSearch
+### openshift_logging_es_allow_external
+If openshift_logging_es_allow_external is set to True, Elasticsearch was exposed as a Route.
+The following steps will achieve the goal.
+* Create a route for ElasticSearch.
+```
+$ oc create -f - << EOF
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: elasticsearch
+  namespace: openshift-logging
+spec:
+  host: elasticsearch.openshift-logging.svc.cluster.local
+  to:
+    kind: Service
+    name: elasticsearch
+  tls:
+    termination: passthrough
```

Comment:
We should continue to use a re-encrypt route if possible.  Doesn't `passthrough` mean the termination ends at the service requiring the ES pod to be able to handle the certs?  @ewolinetz looking at the ansible code seems the destination ca is required for a re-encrypt route.  Is the one used by the operator accessible to an admin?

Reply:
Thanks. Replaced the termination type with reencrypt